platform/jewel/detekt-plugin/src/main/kotlin/org/jetbrains/jewel/detekt/rules/EqualityMembersRule.kt

```diff
@@ -30,7 +31,7 @@ import org.jetbrains.kotlin.psi.psiUtil.findDescendantOfType
  * This rule also supports auto-correction. It can generate the missing or incorrect functions, ensuring they are
  * correctly implemented based on the class's properties.
  */
-class EqualityMembersRule(config: Config) : Rule(config) {
+class EqualityMembersRule(private val config: Config) : Rule(config) {
```

Comment:
Was the addition of the `private val` intended? We are not using it in this scope